frontend/src/views/SubmissionView.vue

```diff
@@ -55,10 +56,15 @@ async function onSubmissionSubmit(data: any) {
   toast.success('Form saved');
 }
 
+const sortPermits = (): void => {
+  permits.value.sort((a: Permit, b: Permit) => a.permitType?.name.localeCompare(b.permitType?.name ?? '') || 0);
+};
+
 onMounted(async () => {
   submission.value = (await chefsService.getSubmission(props.formId, props.submissionId)).data;
   documents.value = (await documentService.listDocuments(props.submissionId)).data;
   permits.value = (await permitService.listPermits(props.submissionId)).data;
+  sortPermits();
```

Comment:
If we're expecting a certain initial delivery format this should be driven by the backend.

Reply:
Put an orderBy in services/chefs.ts